### ISSUE TITLE
Issue with the Singleplayer Auto add

### ISSUE THREAD

**Issue by Michi63-sys** [OverlordZorn/immersion-cigs-rewrite#36]
It doesn't work, is it known?

**Comment by OverlordZorn** [comment]
No its not - what not working exactly?

did you set the addon settings?

**Comment by Michi63-sys** [comment]
Yes i tried it in the editor and yes i did the addon options enable for it on misson and server site, it doesnt add cigarettes or lighter

**Comment by OverlordZorn** [comment]
The Editor is excluded from the "Add Cigs in SP" feature.
Was like this in the original mod. I can add another setting which would enable it in the editor, eventhough i doubt it would be of much use besides simply testing it. The original feature is intended for SP Scenarios and Campaigns.
